src/Sarif.Driver/Sdk/SarifLogger.cs

```diff
@@ -102,7 +102,7 @@ public static ToolInfo CreateDefaultToolInfo(string prereleaseInfo = null)
                             }
                             });
                         }
-                        runInfo.AnalysisTargets.Add(fileReference);
+                        runInfo.FileInfo.Add(fileReference.Uri.OriginalString, new FileReference[] { fileReference });
```

Comment:
this is a breaking change it is actually even required. I see the old OM used IList<T> everywhere.